scripts/metadata_comparison/test/test_extractor.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+import unittest
+from metadata_comparison.extractor import *
+
+class ExtractorTestMethods(unittest.TestCase):
+
+    def read_contents(self, file):
```

Comment:
this seems perhaps more generally useful, if there isn't already something like this among the included batteries

Reply:
It's actually not used until part 2... surprisingly, there isn't a built in function to do this, probably because they want to encourage you to actually error-handle properly. Since this is used in test code, I'm being a little lax.